Makefile

```diff
@@ -24,6 +24,8 @@ terraform-hack-init:
 
 integration-tests: run
 	./scripts/wait-for-bridge.sh
+	@echo "==> Attempting to sourcing .env file"
+	if [ -f .env ]; then set -o allexport; . ./.env; set +o allexport; fi; \
```

Comment:
There's gotta be a way we can export this across all sub shells - could try the top answer from here: https://stackoverflow.com/questions/44628206/how-to-load-and-export-variables-from-an-env-file-in-makefile?

Reply:
So turns out it's even easier than that. https://unix.stackexchange.com/a/348432

```
# Load the environment variables. If this errors review the README.MD and create a .env file as instructed
include .env
export
```

Reply:
FYI @EliiseS this seems to work well as long as you don't `"` variables, might be useful for future stuff. Also let me know if this is something you ruled out and I'm missing a flaw with the approach.

Reply:
I tried that, but it ended up not working with the "feature {}", so I went with the hacky approach.

Reply:
Hmmm wonder if this varies between machines. For me with 

This `Makefile`
```makefile
include .env
export

testenv:
	echo ${PROVIDER_CONFIG_HCL}
	$(shell echo $$PROVIDER_CONFIG_HCL)
```
with this `.env`
```bash
PROVIDER_CONFIG_HCL=features {}
TF_PROVIDER_NAME=azurerm
TF_PROVIDER_PATH=./hack/.terraform/plugins/linux_amd64/
```

outputs:

![image](https://user-images.githubusercontent.com/1939288/89819095-c0f7a100-db42-11ea-8da1-cbad06244214.png)

Double checked and the env isn't in the current bash shell already:

![image](https://user-images.githubusercontent.com/1939288/89819144-d40a7100-db42-11ea-8004-86e9f4bcc1a0.png)